lib/panomosity/xlsx_writer.rb

```diff
@@ -0,0 +1,51 @@
+require 'write_xlsx'
+
+module Panomosity
+  module XLSX_Writer
+    def csv_to_xlsx(input_files, output_file)
```

Comment:
This is actually how I originally wrote it. I think I changed it to take both a string or an array simple because an array with one string felt wrong to me.

Reply:
If the files were handled differently based on if there is one or many, I would totally agree.
But since we do the same thing to it either way, this makes it more difficult to maintain instead.

If I come along later and want to change the writing logic, I would have to know to change it in both places. This design would make me think that one is supposed to be different than the other.